.ci-pipelines/ci-manual-cleanup-script.sh

```diff
@@ -18,7 +18,10 @@ for this_test in $all_tests; do
     echo ""
     echo ">>>>>>>> Making clean for $this_test <<<<<<<<"
     echo ""
-    make -f Makefile_$this_test clean
+    make -f Makefile_$this_test distclean
+
+    # Also remove other output files from the tests
+    rm -f *.m
```

Comment:
One of the tests (I think rosadj) puts its results into a log file with a *.m extension.  This should also be removed when cleaning up manually-generated CI-tests.